coral/src/app/layout/header/ProfileDropdown.tsx

```diff
@@ -0,0 +1,121 @@
+  Typography,
+  Icon,
+  useToast,
+} from "@aivenio/aquarium";
+import user from "@aivenio/aquarium/dist/src/icons/user";
+import logOut from "@aivenio/aquarium/dist/src/icons/logOut";
+import classes from "src/app/layout/header/ProfileDropdown.module.css";
+import { logoutUser } from "src/domain/auth-user";
+
+type MenuItem = {
+  path: string;
+  name: string;
+};
+
+const menuItems: MenuItem[] = [
+  {
+    path: "/myProfile",
+    name: "My profile",
+  },
+  {
+    path: "/tenantInfo",
+    name: "My tenant info",
+  },
+  {
+    path: "/changePwd",
+    name: "Change password",
+  },
+];
+
+const LOGOUT_KEY = "logout";
+function ProfileDropdown() {
+  const toast = useToast();
+  function navigateToAngular(path: string) {
+    window.location.assign(`${window.origin}${path}`);
+  }
+
+  function onDropdownClick(actionKey: string | number) {
+    if (actionKey === LOGOUT_KEY) {
+      toast({
+        message: "You are being logged out of Klaw...",
+        position: "bottom-left",
+        variant: "default",
+        duration: 30 * 1000,
+      });
+      logoutUser().catch((error) => {
+        if (error.status !== 401) {
+          toast({
+            message:
+              "Something went wrong in the log out process. Please try again or contact your administrator.",
+            position: "bottom-left",
+            variant: "danger",
+          });
+          console.error(error);
+        } else {
+          window.location.assign(`${window.origin}/login`);
+        }
+      });
+
+      return;
+    } else {
+      const selectedItem = menuItems[actionKey as number];
+      // selectedItem should never be undefined, this is to support
+      // developers and make sure we can not make a mistake here
+      if (selectedItem === undefined) {
+        console.error(`No item with index ${actionKey} found.`);
+      } else {
+        navigateToAngular(selectedItem.path);
+      }
+    }
+  }
+
+  return (
+    <DropdownMenu
+      onAction={onDropdownClick}
+      maxWidth={300}
+      placement="bottom-right"
+      header={
+        <Box.Flex backgroundColor={"primary-10"} paddingLeft={"l1"}>
+          <Box paddingTop={"l1"} className={"profile-dropdown-header"}>
+            <Typography.Large color="grey-90" className="panel-header-title">
+              User name
+            </Typography.Large>
+            <Typography.Caption color="grey-50">Team</Typography.Caption>
+          </Box>
+          <Icon
+            color={"primary-30"}
+            icon={user}
+            height={80}
+            width={80}
+            className={classes.profileDropdownDeaderIcon}
+          />
+        </Box.Flex>
+      }
+    >
+      <DropdownMenu.Trigger>
+        <button aria-label={"Open profile menu"}>
+          <Icon icon={user} fontSize={"20px"} color={"grey-0"} />
+        </button>
+      </DropdownMenu.Trigger>
+      <DropdownMenu.Items>
+        <DropdownMenu.Section>
+          {menuItems.map((item, index) => {
```

Comment:
I'm wondering if we could somehow have `role="link"` on the `DropdownMenu.Item` which will call `navigateToAngular`... Or if it's desirable. Theu are announced as "menu item" currently, which is correct, and probably it's understandable that "My profile" goes to another page. I'm just wary of things that are links but not links in the markup haha!

Reply:
It does not take a property `role` unfortunately. I'm also not a big fan solving it this way, since it's semantically not correct, but also reduces usability, e.g. opening the links in a new tab etc. But there's not really a way to modify it. I've a list with little things that could be improved for this component in the DS, hopefully there will be a bit time & then I can see if I make a PR.

Reply:
How about rendering a `Link` or `a` inside `DropdownMenu.Item` and not using the `onAction` callback for those links?

Is that even worse semantically? Haha!

Reply:
I tried that, but then it renders a `li` element with role `menuitem`, so screenreader get the information that this is a menu item and can be choosen by clicking CTRL-OPTION-Space -> but doing that doesn't do anything, because this would trigger the action. So they would need to try going inside the menu item (which is not what you would expect) and only there's the link. 

For visual users you still have the hover-behaviour on the whole list item, so it feels clickable. If you click, the menu closes but nothing else happens. You have to really hover over the link to click it. 

And for keyboard users, there's no chance to access the link at at.

Reply:
Ah well, thank you for your work. Hopefully the issues will get resolved upstream :o